### PR TITLE
Container light occlusion

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/space-wizards/space-station-14/pull/687

### CODE REVIEW

**Review by PJB3005 — COMMENTED** [review]
Does this work correctly when you have nested containers? e.g. a box without occlusion inside a backpack?

**Comment by ghost** [comment]
will check when I'm home

**Comment by ghost** [comment]
haven't found a way to break it, and we don't have boxes to fit inside backpacks

File: Robust.Client/GameObjects/Components/Light/PointLightComponent.cs [hunk 1]
```diff
@@ -100,6 +107,7 @@ public bool VisibleNested
         private Color _color = Color.White;
         private Vector2 _offset;
         private bool _enabled = true;
+        private bool _occluded = true;
```
Comment (PJB3005) [inline Robust.Client/GameObjects/Components/Light/PointLightComponent.cs]: This shouldn't default to true.

**Review by PJB3005 — DISMISSED** [review]
I tested this by making the container of a locker occlude.

If you drop the flashlight into the locker and close it, it does not let light through.
If you hold the flashlight in your hand and close it on you, it does let light through.

This should be fixed.

**Comment by ghost** [comment]
unsure how to solve this

**Pull request merged**